test/jdk/java/lang/ClassLoader/getResource/GetResource.java

```diff
@@ -144,26 +142,13 @@ public void testCurrentDirA(List<String> options, String expected) throws Throwa
     private void runTest(Path dir, List<String> options, String expected)
         throws Throwable
     {
-        String javapath = JDKToolFinder.getJDKTool("java");
-
         List<String> cmdLine = new ArrayList<>();
-        cmdLine.add(javapath);
         options.forEach(cmdLine::add);
 
         cmdLine.add("GetResource");
         cmdLine.add(expected);
-
-        System.out.println("Command line: " + cmdLine);
-        ProcessBuilder pb =
-            new ProcessBuilder(cmdLine.stream().toArray(String[]::new));
-
-        // change working directory
-        pb.directory(dir.toFile());
-
-        // remove CLASSPATH environment variable
-        Map<String,String> env = pb.environment();
-        String value = env.remove("CLASSPATH");
-
```

Comment:
Looking into the implementation, it seems that the `CLASSPATH` environment variable is only cleared when `test.noclasspath` == true:

```
        if (noCP) {
            // clear CLASSPATH from the env
            pb.environment().remove("CLASSPATH");
        }
```

(This seems to be contrary to the doc comment on `createTestJavaProcessBuilder` though, which says that _unless_ `test.noclasspath` is true, the env. var will be cleared).

Should this test be run with `-Dtest.noclasspath=true`?

Reply:
I updated the test to remove `CLASSPATH` env var.   We should file an issue for the ProcessTools javadoc.